xds/src/main/java/io/grpc/xds/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The XDS loadbalancer plugin implementation.
+ */
+@io.grpc.ExperimentalApi("https://github.com/grpc/grpc-java/issues/5288")
+package io.grpc.xds;
```

Comment:
"No newline at end of file" as reported by github

Reply:
Done.